ldms/src/sampler/zfs_common/zfs_common.h

```diff
@@ -0,0 +1,41 @@
+
+/* -*- c-basic-offset 8 -*- */
+/* Copyright 2022 Lawrence Livermore National Security, LLC
+ * See the top-level COPYING file for details.
+ *
+ * SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+ */
+
+#ifndef ZFS_COMMON
+#define ZFS_COMMON
+
+#include <libzfs.h>
+
+#ifndef ARRAY_LEN
+#define ARRAY_LEN(a) (sizeof(a) / sizeof(*a))
+#endif
+
+#define DEFAULT_ARRAY_LEN 32
+#define MAX_ZFS_NAME_LEN  256
+
+typedef enum {
+	TOPVDEV_COUNT  = 0,
+	LEAFVDEV_COUNT = 1,
+	ALLVDEV_COUNT  = 2
+} vdevcount_type;
+
+
+typedef struct {
+	vdevcount_type	counttype;
+	uint_t		vdev_count;
+} vdevs_count_;
```

Comment:
```suggestion
} vdevs_count_t;
```

Using the `_t` suffix for a typedef would be the usual convention.